heartbeat/azure-events.in

```diff
@@ -6,56 +6,39 @@
 #	(c) 2018 	Tobias Niekamp, Microsoft Corp.
 #				and Linux-HA contributors
 
-import os, sys, time, subprocess
+import os
+import sys
+import time
+import subprocess
 import json
-import urllib, urllib2, socket
-import logging, syslog
-from enum import Enum
+try:
+		import urllib2
+except ImportError:
+		import urllib.request as urllib2
+import socket
 from collections import defaultdict
 
+OCF_FUNCTIONS_DIR = os.environ.get("OCF_FUNCTIONS_DIR", "%s/lib/heartbeat" % os.environ.get("OCF_ROOT"))
+sys.path.append(OCF_FUNCTIONS_DIR)
+import ocf
+
 ##############################################################################
 
 VERSION = "0.10"
 
-OCF_SUCCESS	= 0
-OCF_ERR_GENERIC = 1
-OCF_ERR_UNIMPLEMENTED = 3
-OCF_ERR_CONFIGURED = 6
-OCF_NOT_RUNNING = 7
-
 attr_globalPullState = "azure-events_globalPullState"
 attr_lastDocVersion  = "azure-events_lastDocVersion"
 attr_curNodeState = "azure-events_curNodeState"
 attr_pendingEventIDs = "azure-events_pendingEventIDs"
 
-default_loglevel = logging.INFO
+default_loglevel = ocf.logging.INFO
```

Comment:
I guess this should be logger.

Reply:
No, I actually changed that to logger but apparently the INFO/DEBUG/etc. constants only exist on the module itself.

Reply:
Oh. Good to check anyways :)